pyxem/signals/reduced_intensity1d.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+"""Signal class for Reduced Intensity profiles as a fucntion of scattering
+vector.
+
+"""
+
+from hyperspy.signals import Signal1D
+import numpy as np
+
+from pyxem.components.reduced_intensity_correction_component import ReducedIntensityCorrectionComponent
+from pyxem.utils.ri_utils import damp_ri_exponential, damp_ri_lorch, \
+    damp_ri_updated_lorch, damp_ri_low_q_region_erfc
+
+
+class ReducedIntensity1D(Signal1D):
+    _signal_type = "reduced_intensity1d"
+
+    def __init__(self, *args, **kwargs):
+        Signal1D.__init__(self, *args, **kwargs)
+
+    def damp_exponential(self, b, inplace=True, *args, **kwargs):
+        """ Damps the reduced intensity signal to reduce noise in the high s
+        region by a factor of exp(-b*(s^2)), where b is the damping parameter.
+
+        Parameters
+        ----------
+        b : float
+            The damping parameter.
+        inplace : bool
+            If True (default), this signal is overwritten. Otherwise, returns a
+            new signal.
+        *args:
+            Arguments to be passed to map().
+        **kwargs:
+            Keyword arguments to be passed to map().
+        """
+
+        s_scale = self.axes_manager.signal_axes[0].scale
+        s_size = self.axes_manager.signal_axes[0].size
+
+        return self.map(damp_ri_exponential, b=b, s_scale=s_scale,
+                        s_size=s_size, inplace=inplace, *args, **kwargs)
+
+    def damp_lorch(self, s_max=None, inplace=True, *args, **kwargs):
+        """ Damps the reduced intensity signal to reduce noise in the high s
+        region by a factor of sin(s*delta) / (s*delta), where
+        delta = pi / s_max. (from Lorch 1969)
```

Comment:
Can we have a proper References section in this docstring with a reference to this please?